src/Apps/Search/Components/GenericSearchResultItem.tsx

```diff
@@ -1,19 +1,26 @@
 import { Box, Flex, Image, Link, Sans, Serif, Spacer } from "@artsy/palette"
 import React, { FC } from "react"
+import styled from "styled-components"
 
 interface GenericSearchResultItemProps {
   imageUrl: string
   name: string
   description?: string
-  index: number
   href: string
   entityType: string
 }
 
+const ImageBox = styled(Box)`
+  height: 70px;
+  width: 70px;
+  margin-right: 20px;
+  background-color: #f8f8f8;
```

Comment:
This should probably inline the styles and use the design system for the margin and color.

Something like, `<Box height={70} width={70} mr={2} bg="black5">...` or whatever

Reply:
Ahh, I was wondering how to set background using a palette defined color. Nice :100: